src/uint.rs

```diff
@@ -955,13 +956,6 @@ macro_rules! construct_uint {
 				Ok(())
 			}
 		}
-
-		#[cfg(feature="std")]
-		impl From<&'static str> for $name {
```

Comment:
there is no reason to remove this

Reply:
If you cannot convert into type without panic then you shoulnd't implement `From`. 

For example `std::u64` doesn't implement it for exactly this reason. I think learning from std lib is a good thing.

Reply:
It's a convenience wrapper used only in tests (that's why it requires `'static` lifetime). We can argue here whether it's a good practice or not, but I won't accept this change cause it is used thousands of times in tests utilizing this library.

Reply:
Ok, then it could be hidden under `#cfg[test]`?

Reply:
panic is not fail, it's panic (unlike typed Result<T, E> which is fail)

`[cfg(test)]` does not propagate through dependencies, and those tests mentioned are not inside this repo exclusively

and anyway, @Pzixel , how is this related to the pr topic?

Reply:
> panic is not fail, it's panic (unlike typed Result<T, E> which is fail)

I'm pretty sure that this quote says conversion must always succeed. If you disagree, what do you think they added this note in the documentation and what does it mean? I'd really like ot make my code better, and such a basic concept worth discussing it.

> how is this related to the pr topic?

I was looking at all things with `std` attribute to deretmine if they could work in no-std environment. And then I realised that this method couldn't work anywhere. 

I probably should create another PR for it, but I strongly think that it should be done. I understand that in may require changes in depended crates, but it's really goes against rust guidelines (as I understand the docs).

Reply:
@debris @NikVolf guys, if you need a convenience wrapper used only in tests, you can make any wrapper you'd like, just call it `parse_unwrap` or anything else, not an ecosystem-wide method with defined semantics.
There's nothing that prevents a user of your library from calling `.into()` that would unexpectedly fail.